packages/storybook/src/nimble/anchor-button/anchor-button-matrix.stories.ts

```diff
@@ -57,7 +57,7 @@ const component = (
     </${anchorButtonTag}>
```

Comment:
Could you document the naming conventions in your PR description somewhere in the repo? The [storybook package CONTRIBUTING](https://github.com/ni/nimble/blob/main/packages/storybook/CONTRIBUTING.md) is probably my first choice. Inline docs on the matrix utility methods could also work.

Reply:
I added a section to the CONTRIBUTING docs. Let me know if you think I missed anything or if anything could be clearer.

Reply:
Just left a 1-character suggestion but otherwise looks good. The "Do" and "Don't" examples are really helpful!